mantidproject/mantid
# Remove leftover pearl_routines system test

Description of work.
This PR removes an old system test for `pearl_routines`. The script itself was removed in an older PR #18594 and as this system test didn't actually use the `pearl_routines` script it wasn't picked up. 

This script duplicated code from `pearl_routines` to check that Mantid methods worked correctly, this is unnecessary as we have unit tests for individual algs and system tests which exercise the system as a whole so it makes sense to remove this system test.

New system tests for `isis_powder` - the replacement are planned once we verify the output of the scripts.

**To test:**
Ensure build passes

Fixes: N/A

*Does not need to be in the release notes.*

---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
